multivariate_grouped_data_plot.Rmd

```diff
@@ -0,0 +1,825 @@
+# How to plot grouped data for multivariate (in Chinese)
+
+Yihui Hu, Liuxin Chen
+
+原始网页 Original website：http://www.sthda.com/english/articles/32-r-graphics-essentials/132-plot-grouped-data-box-plot-bar-plot-and-more/#grouped-continuous-variables
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+
+
+## 1 数据准备
+
+加载所需的软件包并将主题函数theme_pubclean()设置为默认主题：
+
+```{r}
+library(dplyr) 
+library(ggplot2)
+library(ggpubr)
+theme_set(theme_pubclean())
+```
+
+  
+  
+
+## 2 分类变量分组制图
```

Comment:
Hashtag headers should not be followed by numbers since the hashtags will create numbered subheadings.